pkg/setting/setting.go

```diff
@@ -462,7 +462,7 @@ func loadSpecifiedConfigFile(configFile string, masterFile *ini.File) error {
 		}
 	}
 
-	userConfig, err := ini.Load(configFile)
+	userConfig, err := ini.Load(makeAbsolute(configFile, HomePath))
```

Comment:
Why are you making this change?

Reply:
It feels odd the `config` flag path shall be absolute while providing `homepath` is required anyway. Should not that `config` path allowed to be relative?
Might be a PR of its own, actually...